src/5e-SRD-Monsters.json

```diff
@@ -37525,8 +37524,7 @@
         "usage": {
           "type": "recharge after rest",
           "rest_types": ["short", "long"]
-        },
-        "attack_bonus": 0
```

Comment:
For some reason the diff shows this change being under actions, which is incorrect. The change was actually under special abilities.

Reply:
It isn't actually showing that. It's just that Github doesn't know how to handle really big files for expansion.